code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -292,8 +292,12 @@
 	var/mob/living/silicon/ai/unlucky_ais
 	for(var/mob/living/occupant as anything in occupants)
 		if(isAI(occupant))
-			unlucky_ais = occupant
-			occupant.gib() //No wreck, no AI to recover
+			var/mob/living/silicon/ai/AI = occupant
```

Comment:
just `ai`

Reply:
you didnt change the variable name i wanted (though i guess it was wrong). im saying dont capitalize `AI` here

Reply:
should be done now